BlockSettleUILib/BSTerminalMainWindow.cpp

```diff
@@ -1336,6 +1337,10 @@ void BSTerminalMainWindow::setupMenu()
    connect(ui_->testEnvSettings, &QPushButton::clicked, this, [this] {
       promptSwitchEnv(false);
    });
+#else
+   ui_->prodEnvSettings->setVisible(false);
+   ui_->testEnvSettings->setVisible(false);
```

Comment:
Maybe it is better to hide widget that contain both buttons?

Reply:
There is no widget which contains explicitly those two buttons, should I add one?

Reply:
You right, widget shared with login button